packages/node-cli/package.json

```diff
@@ -19,7 +19,7 @@
 	"repository": "https://github.com/oracle/netsuite-suitecloud-sdk/tree/master/packages/node-cli",
 	"main": "./src/suitecloud.js",
 	"preferGlobal": true,
-	"sdkFilename": "cli-2023.2.0.jar",
+	"sdkFilename": "cli-2024.1.0.jar",
 	"nsCompatibleVersion": "2023.2",
 	"sdkDownloadUrl": "https://system.netsuite.com/download/suitecloud-sdk/23.2",
```

Comment:
Please `sdkFilename`, `nsCompatibleVersion` and `sdkDownloadUrl` property values to reference `24.2` future version.

Reply:
Thanks David. Good catch
However, should we instead update to version 24.1? 
If not, I fear all testing and debug would not be working until August when release 24.2 is released